requirements.txt

```diff
@@ -1,16 +1,16 @@
 django>=1.11.0,<1.12.0
 elasticsearch==5.3.0
 enum34==1.1.6
+git+https://github.com/flywheel-io/gears.git@v0.1.4#egg=gears
+ipython
```

Comment:
I'm guessing you want IPython for debugging, but it's such a quick install and I'd rather not bake it into the image.

Reply:
Just seeing where IPython came from. I'm taking it out.